components/Dashboard/DashboardWrapper.tsx

```diff
@@ -43,38 +44,50 @@ interface Props {
   children: React.ReactChild;
 }
 
-const DashboardWrapper = ({ children }: Props): React.ReactElement => (
-  <>
-    <Head>
-      <title>Dashboard | Social care | Hackney Council</title>
-    </Head>
+const DashboardWrapper = ({ children }: Props): React.ReactElement => {
+  const { getConfigValue } = useAppConfig();
 
-    <div className={`govuk-grid-row ${s.outer}`}>
-      <div className="govuk-grid-column-one-quarter">
-        <div className={s.sticky}>
-          <nav>
-            <ul className="lbh-list">
-              {navigation.map((link) => (
-                <NavLink href={link.href} key={link.href}>
-                  {link.text}
-                </NavLink>
-              ))}
+  let navLink;
 
-              <ConditionalFeature name="workflows-pilot">
-                <NavLink
-                  href={process.env.NEXT_PUBLIC_WORKFLOWS_PILOT_URL as string}
-                >
-                  Workflows
-                </NavLink>
-              </ConditionalFeature>
-            </ul>
-          </nav>
-          <MyData />
+  try {
```

Comment:
I would probably do this slightly differently than you have here. The `try` / `catch` is a little odd inside the React component.

Is there an issue if we just inline it further down, like so:

```tsx
<ConditionalFeature name="workflows-pilot">
  <NavLink href={getConfigValue('workflowsPilotUrl') as string}>Workflows</NavLink>
</ConditionalFeature>
```